core/templates/pages/topic-viewer-page/topic-viewer-page.component.html

```diff
@@ -48,17 +48,17 @@
       </topic-info-tab>
     </div>
     <div ng-if="$ctrl.activeTab === 'story'">
-      <stories-list canonical-story-summaries="$ctrl.canonicalStorySummaries"></stories-list>
+      <stories-list [canonical-story-summaries]="$ctrl.canonicalStorySummaries"></stories-list>
```

Comment:
Do we really need `$ctrl` here?

Reply:
I've not migrated topic-viewer-page component so the component is still linked using $ctrl, correct me if I'm wrong!

Reply:
I remember Srijan and me discussed that since `stories-list` is migrated to Angular it would work without $ctrl too here. @srijanreddy98 could you please chime in here?=

Though your argument makes complete sense @orthodoxparadox (just want a confirmation here).

Reply:
I was currently using this for reference from 9673:
![Screenshot 2020-07-07 at 10 51 11 PM](https://user-images.githubusercontent.com/30462390/86818795-70ed7080-c0a4-11ea-867c-97c340b644be.png)
Please excuse the screenshot, I don't know how to link diffs!

Reply:
If the topic viewer page is an ajs component, then you need $ctrl.

Reply:
Yup, got it.